airqo-platform/AirQo-api
# Device registry bug fix

- What does this PR do?
This PR fixes issues/bugs that were present when creating a new device.
- What JIRA task is related to this PR?
The link to the JIRA board task is [here](https://airqoteam.atlassian.net/browse/APA-112?atlOrigin=eyJpIjoiYTIwMmE5YTkwZWYxNGZjYzkxNjg3MzNlZjM4OTFjOTYiLCJwIjoiaiJ9).
- How do I test out this PR?
Git clone this repo `cd AirQo-api/src/device-registry` Then run `npm install` to get all the necessary dependencies Afterwards run `npm run test` to test out the end points using any REST client tool available to you.
- Which end points should are ready for testing?
base URI: https://device-registry-dot-airqo-250220.appspot.com
partial URI: /api/v1/devices
